packages/esm-patient-chart-app/translations/fr.json

```diff
@@ -62,6 +62,9 @@
   "error": "Erreur",
   "errorCancellingVisit": "Erreur en annullant la visite active",
   "errorCopy": "Sorry, there was a problem displaying this information. You can try to reload this page, or contact the site administrator and quote the error code above.",
+  "errorCreatingVisitAttribute": "Could not delete {{attributeName}} attribute",
```

Comment:
Translations should only apply to `en.json`, so please remove the other files from this PR.